src/main/java/frc/robot/Robot.java

```diff
@@ -52,7 +52,7 @@
 
   private final int AUTONOMOUS_BALL_COUNT = 3;
   private final double FIRE_DURATION_SECONDS = 0.5;
-  private final int BARF_TIMER_DURATION = 5;
+  private final int BARF_TIMER_DURATION = 3;
```

Comment:
Please reset the storage count on BARF operation.

Reply:
Fixed